Server crashed, htmlPlugin.options is undefined

### What's wrong?
Webpack 5.18.0 project.
Install 
```
[Cosmos] Using default cosmos config
[Cosmos] See you at http://localhost:5000
[Cosmos] Using default webpack config
[Cosmos] Server crashed...

  (╯°□°)╯︵ ┻━┻

TypeError: Cannot read property 'filename' of undefined
```
[Place where error throw](https://github.com/react-cosmos/react-cosmos/blob/dbe83a74f53de7ae3dd61f822aec56e673f725b9/packages/react-cosmos/src/plugins/webpack/webpackConfig/htmlPlugin.ts#L69)

### Steps to reproduce
1. Create new project with with next webpack [config](https://gist.github.com/Akiyamka/e6cca2ccaa0f3f8384a989d4743acdf3)
Few stupid simple react components <App /> <HelloWorld /> and <Button />, check it work as expected.
2.
```
npm i --D react-cosmos
```
3. add 
```
    "cosmos": "cosmos",
    "cosmos:export": "cosmos-export"
 ```
 to package.json scripts
4.
```
npm run cosmos
```

### Screenshots

![image](https://user-images.githubusercontent.com/6587585/106128186-3e09e780-6170-11eb-98f2-dd231052e98e.png)


### User info

- Server terminal output: 
```
npm run cosmos

> landing@0.0.1 cosmos .../landing
> cosmos

[Cosmos] Using default cosmos config
[Cosmos] See you at http://localhost:5000
[Cosmos] Using default webpack config
[Cosmos] Server crashed...

  (╯°□°)╯︵ ┻━┻

TypeError: Cannot read property 'filename' of undefined
    at isIndexHtmlWebpackPlugin (.../landing/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/htmlPlugin.js:75:37)
    at changeHtmlPluginFilename (.../landing/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/htmlPlugin.js:67:8)
    at .../landing/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/htmlPlugin.js:42:44
    at Array.map (<anonymous>)
    at ensureHtmlWebackPlugin (.../landing/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/htmlPlugin.js:41:20)
    at getPlugins (.../landing/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/devServer.js:137:49)
    at _callee$ (.../landing/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/devServer.js:65:24)
    at tryCatch (.../landing/node_modules/regenerator-runtime/runtime.js:63:40)
    at Generator.invoke [as _invoke] (.../landing/node_modules/regenerator-runtime/runtime.js:293:22)
    at Generator.next (.../landing/node_modules/regenerator-runtime/runtime.js:118:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! landing@0.0.1 cosmos: `cosmos`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the landing@0.0.1 cosmos script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     .../.npm/_logs/2021-01-28T10_31_36_997Z-debug.log
```

- Browser console output: null
- Cosmos version: 5.5.3
- Cosmos config: ["No config is required to star" ](https://github.com/react-cosmos/react-cosmos/tree/master/docs#config)
-  webpack version: 5.18
-  webpack [config](https://gist.github.com/Akiyamka/e6cca2ccaa0f3f8384a989d4743acdf3)

### Additional context

Add any other context about the problem here.

--- a/packages/react-cosmos/src/plugins/webpack/webpackConfig/htmlPlugin.ts
+++ b/packages/react-cosmos/src/plugins/webpack/webpackConfig/htmlPlugin.ts
@@ -66,7 +66,7 @@ function changeHtmlPluginFilename(htmlPlugin: HtmlWebpackPlugin) {
 }
 
 function isIndexHtmlWebpackPlugin(htmlPlugin: HtmlWebpackPlugin) {
-  const { filename } = htmlPlugin.options;
+  const { filename } = htmlPlugin.options || htmlPlugin.userOptions;
   return (
     filename === 'index.html' ||
     typeof filename !== 'string' ||

--- a/packages/react-cosmos/src/plugins/webpack/webpackConfig/htmlPlugin.ts
+++ b/packages/react-cosmos/src/plugins/webpack/webpackConfig/htmlPlugin.ts
@@ -56,7 +56,7 @@ function changeHtmlPluginFilename(htmlPlugin: HtmlWebpackPlugin) {
   if (!isIndexHtmlWebpackPlugin(htmlPlugin)) return htmlPlugin;
 
   const safeOptions = omit(
-    htmlPlugin.options,
+    htmlPlugin.options || htmlPlugin.userOptions,
     'chunks'
   ) as HtmlWebpackPlugin['options'];
   return new htmlPlugin.constructor({

